gridspeccer/core.py

```diff
@@ -107,6 +107,22 @@ def make_figure(name, folder=Path("../fig"), filetype=".pdf"):
     except AttributeError:
         log.warning("Not plotting labels for figure %s", name)
 
+    if independent_plots_filetype:
+        try:
+            _, sub_plots = plotscript.get_gridspec(independent_plots=True)
+        except AttributeError:
+            log.error("When retrieving independent_plots as required by the argument given there was a problem.")
```

Comment:
Maybe also print the retrieved error here.

Reply:
Yes changed. (although, have a look at your error message above for the exact same call)